src/content/general/architecture/capi/index.md

```diff
@@ -0,0 +1,227 @@
+---
+linkTitle: Cluster API architecture
+title: The Giant Swarm Platform Architecture
+description: Architecture overview explaining how our Platform is built and what services do we offer.
+weight: 30
+menu:
+  main:
+    parent: general-architecture
+last_review_date: 2022-09-12
+user_questions:
+  - Do you run a Developer Platform?
+aliases:
+  - /basics/platform-architecture/
+owner:
```

Comment:
```suggestion
owner:
```

No need for aliases here. aliases are created in case a page is moved, to have a redirect from the old URL to the new one.